packages/api/src/controllers/asset.ts

```diff
@@ -47,7 +47,10 @@ const app = Router();
 
 const META_MAX_SIZE = 1024;
 
-function validateAssetMeta(meta: Record<string, string>) {
+function validateAssetMeta( meta: Record<string, string>, asset?:WithID<Asset> ) {
+  meta = _({ ...asset.meta, ...meta })
+            .omit(_.isNull)
```

Comment:
This does not "compile". You need to fix your typescript env and pay attention to the errors it shows. The correct function to call here is `omitBy` (but you also need to test it)
![Screen Shot 2022-08-26 at 17 38 33](https://user-images.githubusercontent.com/1613383/186987195-7a9d086e-3094-4d37-88a4-8773af77682c.png)